.azure/test-linux.yml

```diff
@@ -84,27 +88,27 @@ jobs:
           env:
             SETUPTOOLS_ENABLE_FEATURES: "legacy-editable"
 
+      - ${{ if eq(parameters.installOptionals, true) }}:
+        - bash: |
+            set -e
+            source test-job/bin/activate
+            python -m pip install -r requirements-optional.txt -c constraints.txt
+          displayName: "Install optional packages"
+
+        - bash: |
+            set -e
+            sudo apt-get update
+            sudo apt-get install -y graphviz
+            pip check
+          displayName: 'Install post-install optional non-Python dependencies'
```

Comment:
What does `install post-install` mean? Maybe remove `post-install`?

Reply:
I think it made more sense in the previous structure, and just got copy-pasted.  Removed in f9b5cba48.